src/auth/routes/clients.js

```diff
@@ -0,0 +1,80 @@
+import express from 'express'
+import bcrypt from 'bcrypt'
+import Client from '../models/Client'
+
+const router = express.Router()
+
+router.get('/', (req, res) => {
+  res.render('clientRegister', {
+    message: '',
+  })
+})
+
+router.post('/', (req, res) => {
+  if (!req.body.companyName) {
+    res.render('clientRegister', {
+      companyName: req.body.companyName,
+      appName: req.body.appName,
+      password: req.body.password,
+      message: 'You must provide a company name.',
+    })
+  } else if (req.body.companyName.length < 3 || req.body.companyName.length > 18) {
```

Comment:
Pourquoi cette règle de taille pour le nom de l'organisation ?

Reply:
Because the client_id's length can't be bigger than 40 characters in oAuth2. And at least 3 to avoid the use of common client_id like `A-a`.